deebotozmo/cli.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python3
+
+import asyncio
+import base64
+import configparser
+import itertools
+import json
+import logging
+import os
+import platform
+import sys
+import time
+from dataclasses import asdict
+from functools import wraps
+
+import aiohttp
+import click
```

Comment:
`ModuleNotFoundError: No module named 'click'`

Please add this requirement to setup.py